dspace-api/src/main/java/org/dspace/submit/step/UploadStep.java

```diff
@@ -614,7 +614,7 @@ public int processUploadFile(Context context, HttpServletRequest request,
                 if (configurationService.getBooleanProperty("submission-curation.virus-scan"))
                 {
                     Curator curator = new Curator();
-                    curator.addTask("vscan").curate(item);
+                    curator.addTask("vscan").curate(context, item.getHandle());
```

Comment:
hmm... this looks interesting: https://github.com/DSpace/DSpace/blob/269af71afb602808a14edf822ad658c3895e0a37/dspace-api/src/main/java/org/dspace/curate/service/WorkflowCuratorService.java